tests/link/test_jax.py

```diff
@@ -1371,3 +1371,11 @@ def test_psi():
     out = psi(x)
     fg = FunctionGraph([x], [out])
     compare_jax_and_py(fg, [3.0])
+
+
+def test_log1mexp():
+    x = scalar("x")
```

Comment:
Can you use a vector and test values on the two branches of the `where` statement?

Reply:
updated! Now test 2 values on each branch of the `where` statement